include/treelite/tree.h

```diff
@@ -424,18 +471,25 @@ struct Model {
   ModelParam param;
```

Comment:
`num_feature`, `num_output_group`, `random_forest_flag` and `param` do not depend on template parameters; I think they would be better placed in the base class (`Model`).

Reply:
Fixed in #201